.github/workflows/quality-control.yml

```diff
@@ -43,11 +108,10 @@ jobs:
         run: |
           mkdir build
           cmake -G Ninja . -B build
-          build-wrapper-linux-x86-64 --out-dir ${{ env.BUILD_WRAPPER_OUT_DIR }} ninja -C build
+          build-wrapper-linux-x86-64 --out-dir ${{ env.BUILD_WRAPPER_OUT_DIR }} ninja -C build -j 2
```

Comment:
Same as above.

Reply:
Could you make another commit suggestion here, so I can add it from my phone? 😂

Reply:
Sec

Reply:
```suggestion
          build-wrapper-linux-x86-64 --out-dir ${{ env.BUILD_WRAPPER_OUT_DIR }} ninja -C build -j 0
```